study.md

```diff
@@ -95,7 +97,7 @@ would use to satisfy these requirements.
 In your own words, explain the role of application seed data.
 
 ```md
-<!-- your response here -->
+Seed data intializes the database.
```

Comment:
Not quite or poor wording.  Seeding is for the initial data that will go into your app, such as state names or something.